test/DotNetty.Tests.End2End/End2EndTests.cs

```diff
@@ -60,7 +60,8 @@ public End2EndTests(ITestOutputHelper output)
                 .Option(ChannelOption.TcpNodelay, true)
                 .Handler(new ActionChannelInitializer<ISocketChannel>(ch =>
                 {
-                    ch.Pipeline.AddLast(TlsHandler.Client("dotnetty.com", null, (sender, certificate, chain, errors) => true));
+                    string targetHost = tlsCertificate.GetNameInfo(X509NameType.DnsName, false);
```

Comment:
SslStream needs targetHost which is basically a DNS name so we're pulling that out of cert. It's test cert so it was OK to just hard-code it but is in general better to get info from the cert itself.